src/solver/batchnorm/forward_spatial_single.cpp

```diff
@@ -40,31 +40,32 @@ namespace solver {
 namespace batchnorm {
 
 bool BnFwdTrainingSpatialSingle::IsApplicable(
-    const ExecutionContext&, const miopen::batchnorm::ProblemDescription& problem) const
+    const ExecutionContext&, const miopen::batchnorm::ProblemDescription& bn_problem) const
 {
-    if(problem.GetDirection() != miopen::batchnorm::Direction::ForwardTraining ||
-       problem.GetMode() != miopenBNSpatial)
+    if(bn_problem.GetDirection() != miopen::batchnorm::Direction::ForwardTraining ||
+       bn_problem.GetMode() != miopenBNSpatial)
```

Comment:
This renaming was not superfluous. I believe that the name `problem` in this context is sufficient. Inside _all_ solvers this structure is called exactly this way, and renaming it will only bring chaos.